htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -1274,10 +1274,8 @@ public function update(User $user, $notrigger = 0)
 
 				if (!empty($this->socpeopleassigned)) {
 					$already_inserted = array();
-					foreach (array_keys($this->socpeopleassigned) as $key => $val) {
-						if (!is_array($val)) {	// For backward compatibility when val=id
-							$val = array('id' => $val);
-						}
+					foreach (array_keys($this->socpeopleassigned) as $val) {
+						$val = array('id' => $val);
```

Comment:
$val can be an id xxx (old versions) but can also be array('id' => xxx)
If it is already an array, the PR will transform it into array('id'=>array('id'=>...